auth/data-structures.md

```diff
@@ -0,0 +1,92 @@
+# Data Structures
+
+## Request Params
+
+```typescript
+interface RequestParams {
+  chainId: string;
+  nonce: string;
+  aud?: string;
+  nbf?: string;
+  exp?: string;
+  statement?: string;
+  requestId?: string;
+  resources?: string[];
+}
+```
+
+## Respond Params
+
+```typescript
+interface RespondParams {
+  topic: string;
+  signature: CacaoSignature;
```

Comment:
I think signature may not be enough as requester do not know the whole payload and type.
could we return Cacao instead?